app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -106,9 +106,10 @@
               android:id="@+id/pin_password_input_pin"
               style="@style/TextInputLayout"
               android:layout_width="0dp"
-              android:layout_height="56dp"
+              android:layout_height="wrap_content"
               android:layout_marginTop="2dp"
               android:hint="@{viewModel.profile.isAdmin ? @string/input_pin_password_as_admin : @string/input_pin_password_as_user}"
+              app:errorMessage="@{viewModel.errorMessage}"
```

Comment:
Ah, does TextInputLayout handle the accessibility case for the error field automatically?

Reply:
Yes, I think so because in other cases also we are setting error message to TextInputLayout so I think that it will handle accessibility too